lib/web_ui/lib/src/engine/text_editing/text_editing.dart

```diff
@@ -642,19 +646,19 @@ abstract class DefaultTextEditingStrategy implements TextEditingStrategy {
       _subscriptions[i].cancel();
     }
     _subscriptions.clear();
-    domElement.remove();
-    domElement = null;
+    _domElement?.remove();
```

Comment:
_domElement is not null at this point

Reply:
Maybe it should be `_domElement!.remove()` instead?

Reply:
Done.